helm/osmcha/templates/cron.yaml

```diff
@@ -51,3 +51,47 @@ spec:
             - name: REDIS_URL
               value: {{ .Values.app.api.redis_url }}
           restartPolicy: Never
+
+---
+
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: osmcha-adiff-service-cronjob
+    app: osmcha
+    environment: {{ .Values.config.environment }}
+    release: {{ .Release.Name }}
+spec:
+  schedule: "*/2 * * * *"
```

Comment:
@willemarcel what's the appropriate schedule for this cron job? Is every 2 minutes ok?

Reply:
It's better to use 1 minute. I've updated it.